setup.py

```diff
@@ -14,7 +14,6 @@
     "formic~=0.9b",
     "gitpython~=2.0",
     "schematics~=2.0.1",
-    "python-dateutil~=2.0"
```

Comment:
We could also consider mirroring `botocore`'s version spec, since we do have an explicit dependency on `python-dateutil` added in https://github.com/remind101/stacker/pull/487. Not sure what's best.

Reply:
I say keep it in here, set it to `>=2.0` or something, and make sure botocore is first in the list of dependencies.  That should handle it, right?

Reply:
Isn't botocore already first in the list of dependencies here?